Changelog.md

```diff
@@ -1,5 +1,32 @@
 # Motoko compiler changelog
 
+
+* motoko (`moc`)
+
+* Support explicit, safe migration of persistent data allowing arbitrary
+  transformations on a selected subset of stable variables.
+  Additional static checks warn against possible data loss (#4812).
+
+  As a very simple example:
+
+  ```
+  import Nat32 "mo:base/Nat32";
+
+  (with migration =
+    func (old : { var size : Nat32 }) : { var length : Nat } {
+      { var length = Nat32.toNat(old.size) }
+    }
+  )
+  persistent actor {
```

Comment:
Question: does this only work with `persistent actor` or also with `actor`?

Reply:
Looks like it works too.

Reply:
With both